command/smartContract.go

```diff
@@ -0,0 +1,55 @@
+package command
+
+import (
+	"github.com/rubixchain/rubixgoplatform/client"
+	"github.com/rubixchain/rubixgoplatform/core"
+)
+
+func (cmd *Command) generateSmartContractToken() {
+	smartContractTokenRequest := core.GenerateSmartContractRequest{
+		BinaryCode: cmd.binaryCodePath,
+		RawCode:    cmd.rawCodePath,
+		SchemaCode: cmd.schemaFilePath,
+		DID:        cmd.did,
+	}
+
+	request := client.SmartContractRequest{
+		BinaryCode: smartContractTokenRequest.BinaryCode,
+		RawCode:    smartContractTokenRequest.RawCode,
+		SchemaCode: smartContractTokenRequest.SchemaCode,
+		DID:        smartContractTokenRequest.DID,
+	}
+
+	basicResponse, err := cmd.c.GenerateSmartContractToken(&request)
+	if err != nil {
+		cmd.log.Error("Failed to generate smart contract token", "err", err)
+		return
+	}
+	if !basicResponse.Status {
+		cmd.log.Error("Failed to generate smart contract token", "err", err)
+		return
+	}
+	cmd.log.Info("Smart contract token generated successfully")
+
+}
+
+func (cmd *Command) fetchSmartContract() {
+	smartContractTokenRequest := core.FetchSmartContractRequest{
+		SmartContractToken: cmd.smartContractToken,
+	}
+
+	request := client.FetchSmartContractRequest{
```

Comment:
In This method also ,  line 37 variable declaration not required and seems redundant